secrets/tx_secrets.json.sample

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Is there a README anywhere that explains how to use this?

Reply:
No, but I can add it.  I demonstrate this in a notebook, so maybe there.  However, this code could be called from anywhere.  I think the best place would be in the top level README.